seed/analysis_pipelines/better/pipeline.py

```diff
@@ -108,7 +108,7 @@ def _prepare_analysis(self, property_view_ids, start_analysis=False):
             raise AnalysisPipelineException(
                 f'Analysis configuration is invalid: {"; ".join(validation_errors)}')
 
-        progress_data = ProgressData('prepare-analysis-better', self._analysis_id)
+        progress_data = self.get_progress_data(analysis)
```

Comment:
note to self: update other pipelines to use these new methods on the Pipeline base class

Reply:
donezo